app/models/contentful/static.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'content'
```

Comment:
@agoldstone93 this require is not needed because you are bot subclassing from it and it doesn't exist in this branch. It might fix the issue in the test suite.